src/dataFiles.ts

```diff
@@ -22,13 +36,8 @@ export function findPotentialNestedRoots(
 			)
 		);
 		return [root, ...allDescendants];
-	} else if (
-		DATA_EXTENSIONS.includes(extname(root)) &&
-		!root.includes("sample")
-	) {
-		return [root];
 	} else {
-		return [];
+		return [root];
```

Comment:
returning `[root]` rather than `[]` in the else case here seems like it might change more behavior than we want it to. Specifically I'm not sure `depth` works any more.

 What we can probably do to fix that is to move the `depth` check into the `checkIgnorePath` if statement - 
```js
if (depth <= 0 || checkIgnorePath()) { 
  return []
}
```

Then, we know below that statement we're just left with the standard dir case and the standard non-dir case.